tutorial/creating_packages/handle_sources_in_packages.rst

```diff
@@ -109,6 +113,15 @@ it. Note that we pass the ``strip_root=True`` argument so that if all the unzipp
 contents are in a single folder, all the contents are moved to the parent folder (check
 the :ref:`conan.tools.files.unzip()<conan_tools_files_unzip>` reference for more details).
 
+.. warning::
+
+    Using not immutable source origins, like the HEAD branch, or any other branch that is
```

Comment:
same as earlier, I'd perhaps start the sentence with the expectation, e.g: "it is expected that retrieving the sources in the future produces the same results. Using mutable source origins, like a moving reference in git (e.g HEAD branch), or the URL to a file where the contents may change over time, is strongly discouraged and not supported. Not following this practice will result in undefined behavior likely to cause breakages"

(but otherwise this is good)